app/src/main/java/ayds/lisboa/songinfo/home/view/DateCalculator.kt

```diff
@@ -0,0 +1,78 @@
+package ayds.lisboa.songinfo.home.view
+
+interface DateCalculatorFactory{
+    fun get(releaseDate: String, releaseDatePrecision: String):DateCalculator
+}
+internal object DateCalculatorFactoryImpl : DateCalculatorFactory {
+    private const val year = "year"
```

Comment:
Las constantes se definen en mayúscula
```suggestion
    private const val YEAR = "year"
```